python_modules/dagster-airflow/dagster_airflow/operators.py

```diff
@@ -254,19 +257,33 @@ def __init__(
     def operator_for_solid(
         cls, pipeline, env_config, solid_name, step_keys, dag, dag_id, op_kwargs
     ):
+        tmp_dir = op_kwargs.pop('tmp_dir', DOCKER_TEMPDIR)
+        host_tmp_dir = op_kwargs.pop('host_tmp_dir', seven.get_system_temp_directory())
+
+        if 'storage' not in env_config:
+            raise AirflowException(
+                'No storage config found -- must configure either filesystem or s3 storage for '
```

Comment:
perhaps consolidate this error message somewhere. I will want to use it for dagit reexecution